src/agent/kata-agent.service.in

```diff
@@ -14,6 +14,7 @@ Wants=kata-containers.target
 # from a VM vsock port
 StandardOutput=tty
 Type=simple
+Environment=OCICRYPT_KEYPROVIDER_CONFIG=/etc/ocicrypt_config.json
```

Comment:
I will change this way to follow the old ways, s.t. kata-agent is responsible for putting `ocicrypt_config.json`.